tests/run-make/simd-ffi/rmake.rs

```diff
@@ -56,7 +56,7 @@ fn main() {
             .target(&target)
             .emit("llvm-ir,asm")
             .input("simd.rs")
-            .arg("-Ctarget-feature=-soft-float,+neon,+sse")
+            .arg("-Ctarget-feature=-soft-float,+neon,+sse2,+msa")
             .arg(&format!("-Cextra-filename=-{target}"))
             .run();
```

Comment:
This test will become a complete nightmare to maintain when we start to properly enforce some basic consistency of target features.